csv/js/src/main/scala/fs2/data/csv/PlatformCellEncoders.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2019 Lucas Satabin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package fs2.data.csv
+
+trait PlatformCellEncoders {}
```

Comment:
For the sake of symmetry, don't we want to have the `URI` encode as well?

Reply:
The URI encoder is in the shared `CellEncoder` class but I could move it here if you prefer

Reply:
Oh I didn't see it, right. This looks good, thanks.